airbyte-integrations/bases/base-java/src/main/java/io/airbyte/integrations/base/errors/messages/PostgresErrorMessage.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.base.errors.messages;
+
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_ACCESS_PERMISSION;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_DB_NAME;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_HOST_OR_PORT;
+import static io.airbyte.integrations.base.errors.utils.ConnectionErrorType.INCORRECT_USERNAME_OR_PASSWORD;
+import static io.airbyte.integrations.base.errors.utils.ConnectorName.POSTGRES;
+
+import io.airbyte.integrations.base.errors.utils.ConnectorName;
+
+public class PostgresErrorMessage extends ErrorMessage {
+
+  {
+    CONSTANTS.put("28P01", INCORRECT_USERNAME_OR_PASSWORD);
```

Comment:
These ones are much closer to the SQLSTATE error codes defined in [Postgres Error Codes](https://www.postgresql.org/docs/current/errcodes-appendix.html) but there still seems to be a mismatch. The takeaway I see is that `08006` is actually a general error code for "connection failure"

As for `28P01` the error code is actually mapped as `invalid_password` while `42501` is `insufficient_privilege` which conflicts with the error message of `Invalid credentials: user access denied.`, right?